ister_gui.py

```diff
@@ -1705,12 +1733,17 @@ def _init_actions(self):
         confirm_user.set_action('nouser', confirm_dhcp)
         confirm_user.set_action('Previous', bundle_selector)
         user_configuration.set_action('Previous', confirm_user)
-        user_configuration.set_action('Next', confirm_dhcp)
+        user_configuration.set_action('Next', confirm_dhcp2)
         confirm_dhcp.set_action('dhcp', confirm_installation)
         confirm_dhcp.set_action('staticip', static_ip_config)
         confirm_dhcp.set_action('Previous', confirm_user)
+        confirm_dhcp2.set_action('dhcp', confirm_installation)
+        confirm_dhcp2.set_action('staticip', static_ip_config2)
+        confirm_dhcp2.set_action('Previous', confirm_user)
         static_ip_config.set_action('Previous', confirm_dhcp)
         static_ip_config.set_action('Next', confirm_installation)
+        static_ip_config2.set_action('Previous', confirm_dhcp)
+        static_ip_config2.set_action('Next', confirm_installation)
```

Comment:
The installer is not linear, so this method can be called in a different step x of n. I have to have two instances here so call the same method with different arguments.